packages/ddd/src/utils/update-eslint-dep-constraints.ts

```diff
@@ -27,21 +26,24 @@ export const updateEslintDepConstraints = (
           depConstraints,
           ruleInnerObj.depConstraints
         );
-        ruleInnerObj.depConstraints = [
+        const updatedDepConstraints = [
           ...ruleInnerObj.depConstraints,
           ...depConstraintsWithoutDuplicates,
         ];
+        ruleInnerObj.depConstraints = removeGlobalDepConstraints(
```

Comment:
i think it should be kept and not removed as it acts as a fallback for things that dont fall in any other rule, it should be placed as the last rule in the list (in general a more specific dep constraint should be before a less specific one)